src/Features/SupportDataBinding/BrowserTest.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace Livewire\Features\SupportDataBinding;
+
+use Livewire\Component;
+use Livewire\Livewire;
+use Tests\BrowserTestCase;
+
+class BrowserTest extends BrowserTestCase
+{
+    /** @test */
+    function can_use_wire_dirty()
+    {
+        Livewire::visit(new class extends Component {
+            public $prop = false;
+
+            public function render()
+            {
+                return <<<'BLADE'
+                    <div>
+                        <input dusk="checkbox" type="checkbox" wire:model="prop" value="true"  />
+
+                        <div wire:dirty>Unsaved changes...</div>
+                        <div wire:dirty.remove>The data is in-sync...</div>
+                    </div>
+                BLADE;
+            }
+        })
+            ->assertSee('The data is in-sync...')
+            ->check('@checkbox')
+            ->assertSee('Unsaved changes...')
```

Comment:
@PhiloNL shouldn't here also be a `->assertDontSee('The data is in-sync')`? Just to make this test more robust for the future

Reply:
Thanks! Added.